### DISCUSSION TITLE
Suggestion - MQTT Updates

### DISCUSSION THREAD

**Discussion by CaptainSteubing** [bourquep/weewx-me.teo#50] [Ideas]
Howdy, 

Thanks for developing this skin for Weewx. It's nice and light and works well on mobile devices. 
May I make a suggestion to have it connect to the server via MQTT for instant updates? The WDC & Belchertown skins seem to have this feature.

Thanks!

**Comment by bourquep** [comment]
Hello!

That's an interesting suggestion, I'll look into it in the following weeks!

**Reply by CaptainSteubing** [reply]
Cool, thanks Pascal!

**Comment by danb35** [comment]
Has there been any progress on this?  Or more broadly, can the page be set otherwise to auto-update?  Reload every few minutes or something?  The skin looks great for turning a spare tablet into a weather console, but it seems less useful if it won't stay up-to-date.

**Comment by bourquep** [comment]
It does auto update every 5 minutes via JavaScript. Le 28 juin 2026 à 14:46, danb35 ***@***.***> a écrit :﻿
Has there been any progress on this?  Or more broadly, can the page be set otherwise to auto-update?  Reload every few minutes or something?  The skin looks great for turning a spare tablet into a weather console, but it seems less useful if it won't stay up-to-date.

**Reply by danb35** [reply]
Ah, indeed--I hadn't left it up long enough to have seen that, but now I see that it does.  Live updates with every packet would still be nice, but this way will still show current conditions.  Thanks.
